contents/blog/new-pricing.md

```diff
@@ -0,0 +1,144 @@
+| **1-2m**           | $0.0000500 / event       | $0.0002480 / event                | -80%           |
+| **2-15m**          | $0.0000343 / event       | $0.0001040 / event                | -67%           |
+| **15-50m**         | $0.0000295 / event       | $0.0000655 / event                | -55%           |
+| **50-100m**        | $0.0000218 / event       | $0.0000364 / event                | -40%           |
+| **100-250**        | $0.0000150 / event       | $0.0000187 / event                | -20%           |
+| **∞**              | $0.0000090 / event       | $0.0000100 / event                | -10%           |
+
+## When should I use the new option?
+
+We use it for events where we're only interested in aggregate data, such as for logged-out users on our website.
+
+It could also be useful when:
+
+- You want to track your marketing website and your product, but you don't need detailed information about visitors to your marketing website.
+- You're willing to lose some functionality for lower bills.
+- You want to track API or server events, especially at high volumes. 
+- You run a large e-commerce or content website with millions of users.
+- You're not tracking a high-volume event because it would be prohibitively expensive.
+- You want to use [autocapture](/docs/product-analytics/autocapture) to track events, but have turned it off to save money.
+
+## How do I start using it?
+
+For most people, it only requires a simple config change to include `process_persons: "identified_only"` in the initialization of your JavaScript Web SDK or snippet.
+
+This will track anonymous traffic without person profiles, and start collecting person information any time an identifying action is taken (i.e. using `identify()`, `group()`, setting person properties with `$set`, etc).
+
+You can get more details on this and other use cases, like server-side events, in our [persons documentation](/docs/data/persons).
+
+## What's the downside?
+
+When sending events without person information, you cannot:
+
+- Filter on persons (e.g. an individual user)
+- Create cohorts filtered on person properties (e.g. paying vs non-paying users)
+- Target by person properties for feature flags, A/B tests, and surveys.
+- Receive initial UTM values for tracking marketing campaign performance (only the most recent).
+- View a person’s profile in the app, or query the persons table in our SQL insights.
+- Use [groups functionality](/docs/product-analytics/group-analytics) on standard events (without person profiles).
+
+If you need the above functionality for all your traffic, you don't need to change anything. Continue to send person information with your events.
+
+Person profiles are not all-or-nothing. You can use them on any traffic where you need more detailed information, and skip them when you don't need that detail. Read more about this in our [persons documentation](/docs/data/persons).
+
+## Why are we doing this?
```

Comment:
I feel like this should be higher, or incorporated elsewhere. Currently the flow of sections feels hard to follow.

- Summary
- How much cheaper are new base events? (What's changed)
- When should I use the new option? (Qualifier)
- How do I start using it? (CTA)
- What's the downside? (Caveat)
- Why are we doing this? (Why it's changed)
- FAQ

So we start with what's changed, then qualify it, then give a CTA, then give some caveats, then explain why we've done it.  That feels muddled.

I think it's a clearer story if we start with what's changed and why, then cover the benefits and caveats, then qualify it and give a CTA. 

I feel it's especially important that we qualify the downsides _before_ we tell people how to get started with it.

Reply:
Just so it's 100% clear, what order do you think they should be in?

Reply:
The part I feel most strongly about is that we end with 'How do I start using this?' and cover the downsides before that. Beyond that, I personally feel that 'Why we're doing this' could be expressed elsewhere but I'd defer to you. 

If I was writing the headings would be something like:

- What's changed and why
- Who is this for/why you should use it
- What's the downside?
- How do I get started?

Reply:
Will switch up the order.